# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/reaction/hotfix-makefile.patch
# Copyright (C) 2023 - 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Add some standard Makefile variables to accomodate with the T2 build system.

  - NoTag <notag@t2sde.org>

--- reaction-v1.1.2/Makefile.orig	2024-01-15 17:36:43.970856663 +0100
+++ reaction-v1.1.2/Makefile	2024-01-15 17:37:01.525858069 +0100
@@ -12,7 +12,7 @@
 	$(CC) -s -static ip46tables.d/ip46tables.c -o ip46tables
 
 reaction: app/* reaction.go go.mod go.sum
-	CGO_ENABLED=0 go build -buildvcs=false -ldflags "-s -X main.version=`git tag --sort=v:refname | tail -n1` -X main.commit=`git rev-parse --short HEAD`"
+	CGO_ENABLED=0 go build -buildvcs=false
 
 reaction.deb: reaction ip46tables
 	chmod +x reaction ip46tables
